docs/usage/managed_seed.md

```diff
@@ -1,6 +1,6 @@
 # Register Shoot as Seed
 
-An existing shoot can be registered as a seed by creating a `ManagedSeed` resource. This resource replaces the `use-as-seed` annotation that was previously used to create [shooted seeds](shooted_seed.md). It contains:
+An existing shoot can be registered as a seed by creating a `ManagedSeed` resource. This resource replaces the `use-as-seed` annotation that was previously used to create [shooted seeds](shooted_seed.md), and that is soon going to be deprecated. It contains:
```

Comment:
Hmm isn't it already "deprecated" by this release note from https://github.com/gardener/gardener/pull/3579?

> Starting with Gardener v1.18, the `shoot.gardener.cloud/use-as-seed` annotation is deprecated.

https://github.com/gardener/gardener/blob/master/docs/usage/shooted_seed.md also states that.